packages/api/src/api/auth/slack.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from 'express';
+import { env } from '../../env';
+
+export const slack = async (req: Request, res: Response) => {
+  const linkBase: string =
+    'https://slack.com/openid/connect/authorize?scope=openid%20email%20profile&response_type=code';
+  // Make this an env variable
+  const redirect: string =
+    '&redirect_uri=https://1376-162-92-127-85.ngrok-free.app/api/callback/slack';
+  // Does this need to be env?
+  const client: string = `&client_id=${env.slackClientID}`;
+
+  const fullLink = linkBase + redirect + client;
```

Comment:
Can you move all this up out of the function? There's no reason to keep redeclaring
```suggestion
  const slackAuthBaseUrl: string =
    'https://slack.com/openid/connect/authorize?scope=openid%20email%20profile&response_type=code';
  const queryArgs = new URLSearchParams({
    'redirect_uri': `${env.baseUrl}/api/callback/slack`,
    'client_id': env.slackClientID,
  });

  const fullLink = `${slackAuthBaseUrl}${queryArgs.toString()}`;
```